make_timeline.py

```diff
@@ -198,7 +198,10 @@ def get_goes_x(tstart, tstop):
     p3 = h5.root.data.col('long')
     ok = (tstart < times) & (times < tstop)
     h5.close()
-    return times[ok], p3[ok]
+    if np.count_nonzero(ok) < 2:
+        return np.array([tstart, tstop]), np.array([1e-10, 1e-10])
+    else:
+        return times[ok], p3[ok]
```

Comment:
I can also probably go ahead and change the name of this variable from p3 because I think it is not p3.